run

```diff
@@ -144,6 +144,9 @@ rsync_back() {
     fi
 }
 
+# set GOPRIVATE
```

Comment:
want to expand this comment to explain why we're setting this?

Reply:
done